woocommerce/google-listings-and-ads
# Update product channel visibility styling to match coupons

### Changes proposed in this Pull Request:

Closes #1062.

This PR updates the styling of channel visibility meta box on product page to align with that on coupons page.

### Screenshots:

#### Pending: grey background without solid left border

<img width="1792" alt="Screenshot 2021-12-02 at 11 33 52" src="https://user-images.githubusercontent.com/914406/144397510-4623979b-2f86-4489-a24e-2d304f92f75f.png">

#### Has issues: yellow background with solid left border

<img width="1792" alt="Screenshot 2021-12-02 at 11 34 19" src="https://user-images.githubusercontent.com/914406/144397611-47fff59a-29ec-44d1-8c70-4b31e82aeeaa.png">

#### GLA isn't set up: no background (i.e. white) and no borders

<img width="1792" alt="Screenshot 2021-12-02 at 12 42 40" src="https://user-images.githubusercontent.com/914406/144397643-30ddf79e-26c6-4b39-bf2d-473514423617.png">

### Detailed test instructions:

1. Install Google Listings & Ads
2. Create a new product
  a. Go to `Products` -> `Add New`
  b. Fill in the product name and `Publish` in the right panel
3. In the right panel, check `Channel Visibility` meta box
4. The grey colour background should be displayed, rather than the yellow background


### Changelog entry

> Tweak - Update product channel visibility's styling to match that of coupons channel visibility
